core/spec.md

```diff
@@ -1086,37 +1088,60 @@ Content-Type: application/json; charset=utf-8
 
-The Registry model defines extension attributes, Groups and Resources supported
-by the implementation of the Registry Service. This information will usually
-be used by tooling that does not have advance knowledge of the type of data
-stored within the Registry.
-
-When a new implementation is first created, or a new specification is based
-on the xRegistry specification, then it MAY choose to modify the core
-attributes to fit its needs. The changes allowed are restricted to the
-following:
-- OPTIONAL attributes MAY be defined as REQUIRED
-- The `description` of an attribute MAY be defined
-- `ifvalues` specifications MAY be defined
-
-To indicate a change to a core attribute, the attribute MUST be defined
-as part of the Registry model. Once a Registry instance is created, further
-changes to these core attribute MUST adhere to the rules defined in the
-[Updating the Registry Model](#updating-the-registry-model) section.
-
-Any core attributes that are not defined in the model of an instance of a
-Registry MUST be implicitly defined. In other words, the Registry's model
-consists of the core attributes (as defined by this specification) overlaid
-with the core and extension attributes that are explicitly defined as part of
-its model.
+The Registry model defines the Groups, Resources, extension attributes and
+changes to specification defined core attributes. This information is
+intended to be used by tooling that does not have advance knowledge of the
+structure of the Registry and therefore will need to dynamically discovery it.
+
+To enable support for a wide range of use cases, but to also ensure
+interoperability across implementations, the following rules have been defined
+with respect to how models are defined:
+- Core attributes that are `serverrequired` MUST NOT have this aspect changed
+  to `false`
+- Core attributes that are `readonly` and `serverrequired` MUST NOT have the
+  `readonly` apsect changed to `false`
+- The `name` and `type` aspects of core attributes MUST NOT be changed
+
+To indicate a change to a core attribute, the attribute MUST be defined as part
+of the Registry model. Any core attributes that are not explicitly defined in
+the model of a Registry MUST be implicitly present and be as defined by this
+specification.  In other words, the Registry's model consists of the core
+attributes (as defined by this specification) overlaid with the core and
+extension attributes that are explicitly defined as part of its model.
+
+Note: there is no mechanism defined to delete core attributes from the model.
+
+TODO can an impl reject the use of core OPTIONAL attributes? e.g. description
+
+Once a Registry has been created, implementations MAY choose to limit the
+types of changes made to the model - for example, to ensure backwards
+compatibility of clients or to ensure existing entities remain consistent with
+the defined model.
+
+Implementations are REQUIRED to ensure that after any model changes are made
```

Comment:
Possibly add a comma at the end of the line? I think that would help in terms of readability.